info/v1/container.go

```diff
@@ -395,6 +395,8 @@ type MemoryStats struct {
 
 	Failcnt uint64 `json:"failcnt"`
 
+	Migrate uint64 `json:"memory_migrate"`
```

Comment:
I think that it will be good to prepare the code for reading other files from cpuset directory so maybe struct for values will be better. What do you think?